### PR TITLE
Updates to documentation

### PR DESCRIPTION
**Pull request by Jimvin**
## Review Checklist
- [x] Code contains useful comments
- [x] (Integration-)Test cases added (or not applicable)
- [x] Documentation added (or not applicable)
- [x] Changelog updated (or not applicable)
- [x] Cargo.toml only contains references to git tags (not specific commits or branches)

### CODE REVIEW

File: docs/modules/ROOT/pages/index.adoc [hunk 1]
```diff
@@ -10,5 +10,5 @@ and only works with images from the https://repo.stackable.tech/#browse/browse:d
 
 [source]
 ----
-docker pull docker.stackable.tech/stackable/zookeeper:<version>
+docker pull https://docker.stackable.tech/stackable/zookeeper:<version>
```
Comment (teozkr) [inline docs/modules/ROOT/pages/index.adoc:13]: Docker uses their own reference format, not URIs. They do not have a schema, nor do they map cleanly onto HTTP(S) endpoints.

File: docs/modules/ROOT/pages/config_properties.adoc [hunk 2]
```diff
@@ -1,6 +1,42 @@
...
+        # Set environment variables here
+        envOverrides:
+          MY_ENV_VAR: "foobar"
+----
+
+
 == Configuration Properties
 
 [cols="2,2,2,2,1"]
+>>>>>>> e52f09c4daf8f3a604a7b95e45dbad1fef09789c
```
Comment (teozkr) [inline docs/modules/ROOT/pages/config_properties.adoc:39]: Leftover from a merge conflict?

File: docs/modules/ROOT/pages/index.adoc (hunk 1 above)
Comment (Jimvin) [reply docs/modules/ROOT/pages/index.adoc]: Quite right, I merged in the URL scheme from a previous commit and should not have.

File: docs/modules/ROOT/pages/config_properties.adoc (hunk 2 above)
Comment (Jimvin) [reply docs/modules/ROOT/pages/config_properties.adoc]: Yes! Removed.

**Review by teozkr — APPROVED** [review]
LGTM, but I'll probably need to update #263 for this

**Pull request merged**